src/styles/projects.css

```diff
@@ -0,0 +1,8 @@
+.projects-grid {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(14rem, 1fr));
+  gap: 1rem;
+  max-width: 100%;
+  position: relative;
+  padding: 4px;
+}
```

Comment:
why the global style? don't see any reason this can't be on the new `RecentlyClaimedProjects` component

Reply:
It's used by default-explore-page, filecoin-explore-page, and recently-claimed-projects

Reply:
I refactored project grids into their own component